utils/mdn_changelog_template.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Generates the Kuma Report Template. Requires 3rd-party libraries
+Generates the MDN Changelog Template. Requires 3rd-party libraries
```

Comment:
I think it should be `MDN Development Changelog`, as content changelog is not covered here.

Reply:
From my perspective, "Changelog" implies development, so "Development Changelog" would be redundant. I'm curious what others think. And yes, I don't plan to start summarizing what's going on with MDN content.

Reply:
Vote is 1-1, and since I'm the author, I break the tie.